docs/api-reference/extensions/v1beta1/definitions.html

```diff
@@ -7195,6 +7195,13 @@ <h3 id="_v1_quobytevolumesource">v1.QuobyteVolumeSource</h3>
 <td class="tableblock halign-left valign-top"><p class="tableblock">string</p></td>
 <td class="tableblock halign-left valign-top"></td>
 </tr>
+<tr>
+<td class="tableblock halign-left valign-top"><p class="tableblock">tenant</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Tenant where the Quobyte Volume is placed Default is DEFAULT</p></td>
```

Comment:
Name of Quobyte Tenant that the Volume is part of. Default is DEFAULT.

Here and everywhere below.